github-actions/scan/src/content-type.ts

```diff
@@ -0,0 +1,34 @@
+// SPDX-License-Identifier: MIT
+
+import { SecHubConfigurationModelBuilderData } from './configuration-builder'
+
+export class ContentType {
+
+    static SOURCE = 'source';
+    static BINARIES = 'binaries';
+
+    public static isSource(data: string | undefined): boolean {
+        if (!data) {
+            return false;
+        }
+        return data.toLowerCase() === this.SOURCE;
+    }
+
+    public static isBinary(data: string | undefined): boolean {
+        if (!data) {
+            return false;
+        }
+        return data.toLowerCase() === this.BINARIES;
+    }
+
+    static ensureAccepted(contentType: string): string {
```

Comment:
The naming is odd, sound like a isAccepted()

```suggestion
    static safeAcceptedContentType(contentType: string): string {
```